diff --git a/node_modules/firebase/compat/index.d.ts b/node_modules/firebase/compat/index.d.ts
index fc2bb09..576eeb9 100644
--- a/node_modules/firebase/compat/index.d.ts
+++ b/node_modules/firebase/compat/index.d.ts
@@ -10313,4 +10313,7 @@ declare namespace firebase.firestore {
 }
 
 export default firebase;
-export as namespace firebase;
+
+declare global {
+  var firebase: import("@firebase/app-compat").FirebaseNamespace;
+}
